fabric-key-binding-api-v1/src/client/java/net/fabricmc/fabric/api/client/keybinding/v1/KeyBindingHelper.java

```diff
@@ -41,8 +44,11 @@ private KeyBindingHelper() {
 	 *
 	 * @param keyBinding the keybinding
 	 * @return the keybinding itself
+	 * @throws NullPointerException when {@code null} is passed
```

Comment:
Not sure I agree with adding this to almost every API method. I feel like `@throws` should only be documented if it's something the API user should be aware of.

Reply:
IMO it doesn't hurt to have, and even the JDK does it. Not suggesting to force it though